cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -274,82 +467,27 @@ configure_nvidia_installation_dirs() {
   popd
 }
 
-major_version() {
-  echo "$1" | cut -d "." -f 1
-}
-
-minor_version() {
-  echo "$1" | cut -d "." -f 2
-}
-
-installer_default_download_url() {
-#  if (( $(major_version "${NVIDIA_DRIVER_VERSION}") < 390 )); then
-# Versions prior to 390 are downloaded from the upstream location.
-info "Downloading Nvidia installer from https://us.download.nvidia.com/... "
-echo "https://us.download.nvidia.com/tesla/${NVIDIA_DRIVER_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
-return
-#  fi
-
-#  info "Downloading Nvidia installer from https://storage.googleapis.com/... "
```

Comment:
@kevincvlam You commented out these lines 1.5 years ago, and I don't know where they got moved to in the main repo